src/containers/MyNdla/Learningpath/components/LearningpathStepForm.tsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) 2024-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { Controller, FormProvider, useForm } from "react-hook-form";
+import { useTranslation } from "react-i18next";
+import {
+  Button,
+  FieldErrorMessage,
+  FieldHelper,
+  FieldLabel,
+  FieldRoot,
+  RadioGroupItem,
+  RadioGroupItemControl,
+  RadioGroupItemHiddenInput,
+  RadioGroupItemText,
+  RadioGroupRoot,
+} from "@ndla/primitives";
+import { HStack, styled } from "@ndla/styled-system/jsx";
+import { LearningpathStepDeleteDialog } from "./LearningpathStepDeleteDialog";
+import { GQLMyNdlaLearningpathStepFragment } from "../../../../graphqlTypes";
+import { getFormTypeFromStep, getValuesFromStep } from "../utils";
+
+const ContentWrapper = styled("form", {
+  base: {
+    width: "100%",
+    display: "flex",
+    flexDirection: "column",
+    gap: "medium",
+    padding: "medium",
+    border: "1px solid",
+    borderColor: "stroke.discrete",
+    background: "surface.default.subtle",
+  },
+});
+
+// Temporary placement until forms are merged
+export interface ExternalFormValues {
```

Comment:
Hvor skal disse egentlig ligge? vurdere å lage en types.ts fil og legge det der?

Reply:
De kommer til å ligge i vær form fil så ligger her kun for å unngå feilmelding i fila og typene som brukes under.